index.html

```diff
@@ -42,7 +44,7 @@ <h3 class="profession">A Web Developer</h3>
         <div class="design"><img src="./images/design.png" alt="design" /></div>
```

Comment:
Kindly. Layout images should be placed in the background of your site, I recommend using background-image.

Reply:
please can you clarify

Reply:
As requested by the previous reviewer, kindly implement the background design using CSS background image. This is because it's not part of the content and only a design. Read more about [when to use CSS background image](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html).

Reply:
I am unable to make it a background image, please help